### PR TITLE
net: openthread: Adding snoop entries kconfig.

### PR DESCRIPTION
**Pull request by canisLupus1313**
Adding missing `CONFIG_OPENTHREAD_TMF_ADDRESS_CACHE_MAX_SNOOP_ENTRIES` to thread kconfig file.

### CODE REVIEW

**Comment by canisLupus1313** [comment]
@arnulfrupp FYI

File: subsys/net/l2/openthread/Kconfig [hunk 1]
```diff
@@ -289,6 +289,13 @@ config OPENTHREAD_TMF_ADDRESS_CACHE_ENTRIES
 	help
 	  The number of EID-to-RLOC cache entries.
 
+config OPENTHREAD_TMF_ADDRESS_CACHE_MAX_SNOOP_ENTRIES
+	int "The maximum number of EID-to-RLOC cache entries"
```
Comment (pdgendt) [inline subsys/net/l2/openthread/Kconfig:293]: I think it makes sense to add a default value?

```suggestion
	int "The maximum number of EID-to-RLOC cache entries"
	default 2
```

Comment (canisLupus1313) [reply subsys/net/l2/openthread/Kconfig:293]: fixed.

**Review by edmont — APPROVED** [review]

**Review by pdgendt — APPROVED** [review]

**Review by rlubos — APPROVED** [review]

**Review by carlescufi — APPROVED** [review]

**Comment by arnulfrupp** [comment]
@canisLupus1313 Tanks!

**Pull request merged**